Makefile

```diff
@@ -81,7 +84,31 @@ generate-embeddedyamls: pkg/subctl/operator/common/embeddedyamls/yamls.go
 pkg/subctl/operator/common/embeddedyamls/yamls.go: pkg/subctl/operator/common/embeddedyamls/generators/yamls2go.go $(shell find deploy/ -name "*.yaml") vendor/modules.txt
 	go generate pkg/subctl/operator/common/embeddedyamls/generate.go
 
-.PHONY: $(TARGETS) test validate build ci clean generate-embeddedyamls operator-image
+# generate-clientset generates the clientset for the Submariner APIs
+# It needs to be run when the Submariner APIs change
+generate-clientset:
+	go get k8s.io/code-generator@kubernetes-1.14.1
+	$${GOPATH}/pkg/k8s.io/code-generator@v0.0.0-20190311093542-50b561225d70/generate-groups.sh \
+		client,deepcopy \
+		github.com/submariner-io/submariner-operator/pkg/client \
+		github.com/submariner-io/submariner-operator/pkg/apis \
+		submariner:v1alpha1
+
+# generate-operator-api updates the generated operator code
+# It needs to be run when the CRDs or APIs change
+generate-operator-api:
+	operator-sdk generate k8s
+	operator-sdk generate openapi
+
+preload-images:
+	source $(SCRIPTS_DIR)/lib/debug_functions; \
+	source $(SCRIPTS_DIR)/lib/deploy_funcs; \
+	set -e; \
+	for image in submariner submariner-route-agent submariner-operator lighthouse-agent submariner-globalnet lighthouse-coredns; do \
+		import_image quay.io/submariner/$${image}; \
+	done
```

Comment:
This one I'd actually leave in place, but whatever you think makes sense..

Reply:
I quite like having `scripts` pretty much empty ;-).